packaging/debian/rules

```diff
@@ -11,7 +11,7 @@ clean:
 	dh_clean
 	if [ -e build ] ; then rm -rf build ; fi ;
 
-install: build
+install:
```

Comment:
Hi @qix-. Thanks for taking a look at this change. You should stop by http://slack.rethinkdb.com #engineering if you would like to get more involved.

`dpkg-buildpackage` calls both targets separately, which currently causes rethinkdb to be built twice when packaging. See http://man7.org/linux/man-pages/man1/dpkg-buildpackage.1.html